plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationPuckManager.kt

```diff
@@ -48,11 +48,17 @@ internal class LocationPuckManager(
   private val onLocationUpdated: ((Point) -> Unit) = {
     lastLocation = it
     if (settings.locationPuck is LocationPuck3D) {
-      val latitude =
-        if (delegateProvider.mapProjectionDelegate.getMapProjection() == MapProjection.Globe) {
-          delegateProvider.mapCameraManagerDelegate.cameraState.center.latitude()
-        } else it.latitude()
-      lastMercatorScale = mercatorScale(latitude)
+      delegateProvider.getStyle { style ->
+        val projectionProperty = style.getStyleProjectionProperty("name")
+        val isGlobeProjection =
+          projectionProperty.kind == StylePropertyValueKind.CONSTANT &&
+            (projectionProperty.value.contents as String).uppercase() == "GLOBE"
```

Comment:
Is there a constant for GLOBE?

Reply:
ProjectionName.GLOBE right

Reply:
we don't have dependency on `style-extension` here so we're using raw values